src/services/__tests__/notificationService.test.ts

```diff
@@ -100,7 +101,7 @@ describe("NotificationService#createOrUpdateInstallation", () => {
     expect(res).toEqual({
       apply: expect.any(Function),
       kind: "IResponseSuccessJson",
-      value: "ok"
+      value: { message: "ok" } as SuccessResponse
```

Comment:
is the cast mandatory here ?

Reply:
No, it's not actually. Removed in commit [4bae737]